packages/browser/src/tracing/request.ts

```diff
@@ -394,6 +395,10 @@ export function xhrCallback(
     );
   }
 
+  if (client) {
+    client.emit('outgoingRequestSpanStart', span, { body: getGraphQLRequestPayload(sentryXhrData.body as string) });
```

Comment:
m: Similar to what I mentioned for the breadcrumb hook, let's avoid using any graphql methods in here. Instead, let's do something like this:

```ts
client.emit('outgoingRequestSpanStart', span: Span, handlerData: HandlerDataXhr | HandlerDataFetch)
```

this again holds either the `args` of the fetch request, or the `xhr` object, from which we can pull the bodies in the integration the same as for the breadcrumbs.